dataland-frontend/tests/component/components/general/DataPointFormField.cy.ts

```diff
@@ -0,0 +1,44 @@
+import DataPointFormField from "@/components/forms/parts/kpiSelection/DataPointFormField.vue";
+
+describe("Component test for DataPointFormField", () => {
+  it("Unit field should be visible when options are defined and Quality field should be NA if the value field has no value", () => {
+    cy.mountWithPlugins(DataPointFormField, {
+      data() {
+        return {};
+      },
```

Comment:
Why is the `data()` being emtpied for the test?

Reply:
I will remove it with next commit

Reply:
👍